### ISSUE TITLE
When using the Forge version, recipes still work

### ISSUE THREAD

**Issue by StormDragon-64** [nvb-uy/Item_Obliterator#21]
I'm using the Forge 1.19.2 version of this mod, but not all of the mod's functions work. 
After putting an item id into the blacklist, relaunching the game, and then putting the item into your inventory, the item gets deleted immediately as expected. However, the item still shows up in EMI and the recipe still works. This means that if someone  tried to craft the item, it would disappear immediately, causing all of the items they just used to craft the item to disappear.
I tried adding the item to the recipes only section of the blacklist, but the recipe still worked.

**Comment by StormDragon-64** [comment]
Ok, so it turns out the recipe part does work fine on 1.7.0, I just originally tested using the latest version for 1.19.2 Forge on CurseForge, which is 1.6.2 instead. Also, the EMI bug still applies.

**Comment by FireWolf700** [comment]
Same issue here, even going back to 1.6.1, so no clue what might be causing it.

**Comment by StormDragon-64** [comment]
The items still showing up in EMI part of this issue could be fixed by porting 2.0.0 back to 1.19.2 forge, but I can't be sure if the recipe problem would be fixed by backporting it alone.
For now I have been using empty recipes in an included datapack in my modpack, but that means that I have to make a new recipe for every item I want to disable, which is kind of annoying.

**Comment by Quadruplay** [comment]
same thing happens to me on 1.20.1

**Comment by nvb-uy** [comment]
> The items still showing up in EMI part of this issue could be fixed by porting 2.0.0 back to 1.19.2 forge, but I can't be sure if the recipe problem would be fixed by backporting it alone. For now I have been using empty recipes in an included datapack in my modpack, but that means that I have to make a new recipe for every item I want to disable, which is kind of annoying.

I'll look into fixing this with the 2.0.0 backport

**Comment by alarmgoose** [comment]
Same thing happening to me still. Has this been fixed?

**Comment by StormDragon-64** [comment]
No, 2.0.0 still hasn't been backported yet.

**Comment by Droid-B1045** [comment]
Doesnt work for me neither.. Example Configuration is pretty lacking
```
{
  "blacklisted_items": [ 
better_weaponry:copper_armor_boots, 
better_weaponry:copper_armor_leggings, 
better_weaponry:copper_armor_chestplate, 
better_weaponry:copper_armor_helmet,
better_weaponry:copper_sword,
    "//Items here will be unusable completely",
    "examplemod:example_item" 
 ],
```

**Comment by nvb-uy** [comment]
Fixed on latest, v2.2.0 was ported into forge. 1.19.2 version should be available in a few hours as it's under manual review for some reason

**Comment by nvb-uy** [comment]
1.19.2 is available

https://modrinth.com/mod/item-obliterator/version/7wB98NiB
